udata/harvest/backends/dcat.py

```diff
@@ -71,8 +69,11 @@ def inner_harvest(self):
             lambda page_number, page: self.process_datasets(page_number, page),
         )
 
-        # TODO call `walk_graph` with `process_dataservices`
-
+        graphs = self.walk_graph(
```

Comment:
```suggestion
        graphs += self.walk_graph(
```
?

Reply:
No it's the same graphs, I should only save the first one or the second one for debug (do not need both)